### DISCUSSION TITLE
Getting Roles from Keycloak for jhipster 8.5.0 no more working

### DISCUSSION THREAD

**Discussion by djibythiaw** [jhipster/generator-jhipster#26324] [Q&A] [answered]
Hello,
I upgrade my project to use the last version of jhipster (8.5.0) but I face an issue. I can't no more getting the roles for the user. In previous version, I can retrieve the role in the authorities like ROLE_ADMIN, ROLE_USER but with this last version, I am having something like "authorities" : [ "SCOPE_openid", "SCOPE_offline_access", "SCOPE_email", "SCOPE_profile" ] when calling the endpoint /api/account.
Is there config to change in Keycloak something like that ?
Thx.

**Comment by mraible** [comment, marked as answer]
Spring Security has always added the scopes as authorities, but we've been stripping them out for years. You could change the logic in [`SecurityConfiguration.java`](https://github.com/jhipster/generator-jhipster/blob/main/generators/server/templates/src/main/java/_package_/config/SecurityConfiguration_imperative.java.ejs#L330) if you want them included.

**Reply by djibythiaw** [reply]
Thank you. I changed the SecurityConfiguration and it's working now.
